src/signals/triage_signals.R

```diff
@@ -150,11 +156,27 @@ approve_signals <- function(df, fn_signals, test) {
     "DELETE: Delete the campaign content and `output/{indicator}/signals.parquet`",
     "file so you can recreate later.\n",
     "ARCHIVE: Delete the email campaign, but move the alert to `output/signals.parquet`\n",
-    "Any other input: Do nothing, so you can decide later.\n",
+    "DO_NOTHING: Do nothing, so you can decide later.\n",
     sep = ""
   )
 
-  user_command <- readline(prompt = "Your command: ")
+  user_command <- readline(prompt = "Your command: ")}
+  if (user_command %in% c("APPROVE", "DELETE")) {
```

Comment:
I would also add the confirmation for archiving. It's not as critical but it does delete the draft campaign that would go out to users and only maintains the internal campaign.